examples/Toast.placement.jsx

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { useToaster, Button } from '@itwin/itwinui-react';
+
+export default () => {
+  const toaster = useToaster();
+
+  React.useEffect(() => {
+    toaster.setSettings({
+      placement: 'bottom-end',
```

Comment:
This only shows one `placement`. Maybe we should render a `<Select>` to showcase all placements?

Reply:
I have added a `Select` for placements.

Reply:
Follow-up suggestion: You can remove the `useEffect` and the `selectedPlacement` state. Instead, you can directly call `toaster.setSettings` inside `onChange`. See https://react.dev/learn/you-might-not-need-an-effect#sharing-logic-between-event-handlers

Reply:
I have removed the useEffect.